### ISSUE TITLE
JS error in console when running Webpack development build for local Wagtail development

### ISSUE THREAD

**Issue by nickmoreton** [wagtail/wagtail#6877]
I'm following the instructions for setup on both Mac docker desktop and ubuntu server and noticed the Richtext fields have no visible content in admin wherever they are used.

Just wondering if anyone else is seeing this?

I can see this when editing the home page of the bakery demo.

If I open another terminal and execute bash on the front end container run `npm run build` the content is visible but when running `npm start` it disappears again.

**Comment by thibaudcolas** [comment]
Hey @nickmoreton, thank you for reporting this! I have transferred the issue over to Wagtail, as I believe it’s a problem regardless of which dev setup you’re using.

Looking through our recent tooling changes (773de3eded6ecdce61c52fd6464b5997a6cc8699) and the error message I’m 99% sure the issue is with the recent update to expose-loader (https://github.com/webpack-contrib/expose-loader/blob/master/CHANGELOG.md#100-2020-06-23), which has the following breaking change:

> do not override existing exposed values in the global object by default, because it is unsafe, please read

>By default loader does not override the existing value in the global object, because it is unsafe. In development mode, we throw an error if the value already present in the global object. But you can configure loader to override the existing value in the global object using this option.

---

This seems to be a problem because we have multiple Webpack entry points that expose React, but I don’t know why that would be the case.

**Comment by lb-** [comment]
I have not seen this error in any client JS dev for at least the last 6 months, it looks like this was fixed as part of general upgrades here 7d7a03f9f2110decb51090631e6f3087600633ad / #7876 

I have tested `npm run build` and `npm start` and not seen this issue.

**Comment by lb-** [comment]
Here is my `npm run build` output for reference

```
➜ npm run build

> wagtail@1.0.0 build
> webpack --config ./client/webpack.config.js --mode production

assets by path wagtail/admin/static/wagtailadmin/ 1.75 MiB 70 assets
assets by path wagtail/images/static/wagtailimages/ 52.9 KiB 12 assets
assets by path wagtail/contrib/ 24.7 KiB 8 assets
assets by path wagtail/documents/static/wagtaildocs/js/*.js 10.1 KiB
  asset wagtail/documents/static/wagtaildocs/js/add-multiple.js 2.86 KiB [compared for emit] [from: wagtail/documents/static_src/wagtaildocs/js/add-multiple.js] [copied] [minimized]
  asset wagtail/documents/static/wagtaildocs/js/document-chooser-modal.js 2.78 KiB [compared for emit] [minimized] (name: document-chooser-modal)
  + 2 assets
assets by path wagtail/snippets/static/wagtailsnippets/js/*.js 4.56 KiB
  asset wagtail/snippets/static/wagtailsnippets/js/snippet-chooser.js 2.36 KiB [compared for emit] [minimized] (name: snippet-chooser)
  asset wagtail/snippets/static/wagtailsnippets/js/snippet-chooser-telepath.js 2.2 KiB [compared for emit] [minimized] (name: snippet-chooser-telepath)
asset wagtail/search/static/wagtailsearch/js/query-chooser-modal.js 893 bytes [compared for emit] [from: wagtail/search/static_src/wagtailsearch/js/query-chooser-modal.js] [copied] [minimized]
asset wagtail/embeds/static/wagtailembeds/js/embed-chooser-modal.js 358 bytes [compared for emit] [from: wagtail/embeds/static_src/wagtailembeds/js/embed-chooser-modal.js] [copied] [minimized]
asset wagtail/users/static/wagtailusers/js/group-form.js 244 bytes [compared for emit] [from: wagtail/users/static_src/wagtailusers/js/group-form.js] [copied] [minimized]
runtime modules 123 KiB 289 modules
orphan modules 527 KiB [orphan] 74 modules
javascript modules 2.11 MiB
  modules by path ./node_modules/ 1.47 MiB 257 modules
  modules by path ./client/src/ 649 KiB 119 modules
  modules by path ./wagtail/ 300 bytes
    modules by path ./wagtail/admin/static_src/wagtailadmin/scss/ 200 bytes 4 modules
    modules by path ./wagtail/contrib/ 100 bytes 2 modules
  external "jQuery" 42 bytes [built] [code generated]
css modules 234 KiB
  modules by path ./wagtail/admin/static_src/wagtailadmin/scss/ 230 KiB 4 modules
  modules by path ./wagtail/contrib/ 1.99 KiB
    css ./node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[3].use[1]!./node_modules/postcss-loader/dist/cjs.js??ruleSet[1].rules[3].use[2]!./node_modules/sass-loader/dist/cjs.js!./wagtail/contrib/styleguide/static_src/wagtailstyleguide/scss/styleguide.scss 400 bytes [built] [code generated]
    css ./node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[3].use[1]!./node_modules/postcss-loader/dist/cjs.js??ruleSet[1].rules[3].use[2]!./node_modules/sass-loader/dist/cjs.js!./wagtail/contrib/typed_table_block/static_src/typed_table_block/scss/typed_table_block.scss 1.6 KiB [built] [code generated]
  css ./node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[3].use[1]!./node_modules/tippy.js/dist/tippy.css 1.38 KiB [built] [code generated]
```
```
wagtail on   (git)-[pr/9172-chore-docs-screenshots|rebase-i]- [$] is 📦 v1.0.0 via ⬢ v16.17.0 took 1m 28s 
```
